core/run_ctest/show_pset.py

```diff
@@ -0,0 +1,13 @@
+import sys, json
```

Comment:
Very minor: maybe add #!/usr/env/python at the start so you can directly run the file like `show_ptest.py foobar` instead of `python3 show_ptest.py foobar`

Reply:
Thanks! But I'm worried that python will be installed in different directories for different computers.

Reply:
`#!/usr/env/python3` (or perhaps `/usr/bin/env/python3`) should work on any reasonably system. And I believe even if the shebang does not work, you don't lose anything coz you can still call `python3` explicitly.

Reply:
I see the new version has been pushed.